components.go

```diff
@@ -4,13 +4,20 @@ import (
 	"fmt"
 	"strings"
 	"time"
+
+	"github.com/teambition/rrule-go"
 )
 
 // Calendar is the top-level iCalendar object.
 type Calendar struct {
 	*Component
 }
 
+// Returns the Recurrence Set for this component.
```

Comment:
Nit: GoDoc comments always need to start with the name: "RecurrenceSet returns the recurrence set for this component.".